core/src/test/java/com/linecorp/armeria/server/file/HttpFileServiceTest.java

```diff
@@ -84,7 +84,7 @@ protected void configure(ServerBuilder sb) {
             sb.serviceUnder(
                     "/uncached/fs/",
                     HttpFileServiceBuilder.forFileSystem(tmpDir)
-                                          .maxCacheEntries(0)
+                                          .entryCacheSpec("off")
```

Comment:
Once undeprecated, the `entryCacheSpec("off")` calls in this PR could be reverted for simplicity.

Reply:
I will revert code